pymc4/tests/test_autoname.py

```diff
@@ -0,0 +1,30 @@
+"""
+Tests for PyMC4 auto naming.
+"""
+
+from .. import model
+
+import pymc4 as pm
+from pymc4 import Normal
+
+def create_rvs(**kwargs):
+    return pm.Normal(0, 1, name="test")
+
+
+@pm.model(auto_name=True)
```

Comment:
This could be placed in fixture to to keep global scope clean

Reply:
Actually that doesn't work here as we can't parse AST nested functions. I added comments to explain that.